react/tic-tac-toe/src/constants/propsTypes.js

```diff
@@ -0,0 +1,11 @@
+import { shape, string, number } from 'prop-types';
+
+export const matchesPropsTypes = shape({
+  id: number,
+  createdAt: string,
```

Comment:
Try to avoid snake case, a posible solution is create a constants for this something like:

```
const players = {
  playerOne: 'player_one',
  playerTwo: 'player_two',
}
```
and use this constants when you need player_one and player_two